language-examples/remedial-js/11.0-event-qeue/README.md

```diff
@@ -0,0 +1,44 @@
+
+# 11.0 - Example Event Qeue
+
+# Purpose:
+
+  - An overview of how the browser event queue works
+  - What it means to post something to the event queue
+  - How user interactions are also processed through the event queue
+
+### Overview
+
+Running applications in a web browser involves a collection of technologies, such as a JavaScript Engine (such as Chrome’s V8), a collection of Web API’s (such as the DOM), and the Event Loop and Event Queue.
+
+### What exactly does the JavaScript Engine do?
+
+It’s job is to go through all the lines of JavaScript in an application and process them one at a time. This means that JavaScript is single-threaded. The main repercussion of this is that if you’re running a line of JavaScript that happens to take a long time to return, then all the code after that will be blocked.
+
+**So how does javascript engine know how to process a single line of javascript at a time?**
+
+It uses a call stack. You can think of a call stack like entering an elevator — the first person who enters the elevator is the last person to exit the elevator, whereas the last person to enter is the first to exit.
```

Comment:
You've provided a metaphor for the call stack, but failed to mention that it is the stack of functions being executed at a given time.